agent/xds/delta_test.go

```diff
@@ -1156,7 +1156,7 @@ func TestServer_DeltaAggregatedResources_v3_ACLEnforcement(t *testing.T) {
 			case err := <-errCh:
 				if tt.wantDenied {
 					require.Error(t, err)
-					require.Contains(t, err.Error(), "permission denied")
+					require.Contains(t, err.Error(), "Permission denied")
```

Comment:
Does this casing change here mean that older consul clients and servers that do `string.Contains` checks will fail to handle denies when you are straddling versions?

Reply:
Hmm. That's not something I'd considered. Wondering if I should back the change out for the xds code for now.

Reply:
Do we have any known cases where the message string is tested for "permission denied" rather than using the status code codes.PermissionDenied? I spent some time code spelunking but didn't really find much either

Reply:
Sorry in my initial skim I neglected to really notice this is not the _consul_ perm denied used all over acls, this is likely fine then.